cf/data/data.py

```diff
@@ -2256,6 +2253,9 @@ def can_compute(self, functions=None, log_levels=None, override=False):
                 False.
 
         """
+        # TODODASKAPI - this method is premature - needs thinking about as part
```

Comment:
These methods marked `TODODASKAPI` are API-related and therefore can probably be bumped to be considered instead as part of work to do after the Dask migration release, i.e. tranform to plain `TODO`s that we can keep track of via a dedicated Issue. I'll note that on the main Issue #433.